core/common/src/rpc/web/WebAppRpcProtocol.ts

```diff
@@ -33,6 +33,7 @@ export interface HttpServerRequest extends BackendReadable {
   headers: { [header: string]: string | string[] | undefined };
   rawHeaders: string[];
   trailers: { [key: string]: string | undefined };
+  trailersDistinct: NodeJS.Dict<string[]>;
```

Comment:
Additions of `trailersDistict` and `headersDistinct` to public `HttpServerRequest` interface can be considered a breaking change according to our policy. 

Reason for this is the fact that `HttpServerRequest` is a copy of `IncomingMessage` class from Node's `http` library. Without adding new properties, we would need to do type casting or something similar.

Similar thing was also done while adding Node 18 support (see _WebAppRpcProtocol.ts_ file changes in #4295).

Reply:
@swbsi

Reply:
Sadly, no. This will cause build errors as it will become incompatible with Node's `IncomingMessage`.